### ISSUE TITLE
SSH to named servers

### ISSUE THREAD

**Issue by artpik** [yuvipanda/jupyterhub-ssh#28]
It would be great to be able to connect with SSH not only to the "main" server, but also to the named servers as well:
https://jupyterhub.readthedocs.io/en/latest/api/app.html#jupyterhub.app.JupyterHub.allow_named_servers

**Comment by stubclan** [comment]
@artpik found any solutions/ alternatives? Looking for the same thing

**Comment by artpik** [comment]
@stubclan no, users have to use SSH to the "main" server only or use a terminal in JupyterHub. SSH to the named servers would be a great feature.

**Comment by stubclan** [comment]
@artpik I actually successfully managed to connect to named servers. I can share the code if you want.. Github isn't letting me actually create a MR so not sure if we can put it there but yea

**Comment by artpik** [comment]
@stubclan Wow, really? The code is in your fork, right? https://github.com/stubclan/jupyterhub-ssh/commits/main/jupyterhub_ssh/__init__.py
It is so nice of you.

**Comment by stubclan** [comment]
@artpik yes you found it. Its just sth small I threw together.

Next I'm looking to see if we can specify what type of server while spawning directly via ssh but I'm not seeing any jupyterhub api support so idk

**Comment by artpik** [comment]
@stubclan Thanks a lot! It is definitely very helpful. I will test it.
